src/Bones.cpp

```diff
@@ -115,155 +92,103 @@ class GLTFModelManager : public IModelManager {
       return false;
     }
 
-    m_keyframes = GetFloat(GetAccessorByIndex(0));
+    LoadAnimationData();
```

Comment:
I think you can just call the three functions here instead of having `LoadAnimationData`

Reply:
Resolved with latest commit